chrome/content/zotero/preferences/preferences_proxies.js

```diff
@@ -56,6 +56,14 @@ Zotero_Preferences.Proxies = {
 	
 	
 	/**
+	 * Enables UI buttons when proxy is selected
+	 */
+	enableProxyButtons: function () {
+		document.getElementById('proxyTree-edit').disabled = undefined;
+		document.getElementById('proxyTree-delete').disabled = undefined;
```

Comment:
Done. The original code used `undefined`, so I initially kept it. Probably saves a bit or two, but agree that `false` is clearer.